test/test_vumps.jl

```diff
@@ -38,15 +38,15 @@ using Random
 
   for multisite_update_alg in ["sequential", "parallel"],
     conserve_qns in [true, false],
-    nsites in [2, 3, 4],
+    nsites in [1, 2, 3, 4],
     time_step in [-Inf, -0.5]
 
-    if conserve_qns
-      # Flux density is inconsistent for odd unit cells
-      isodd(nsites) && continue
-    end
+    #if conserve_qns
+    # Flux density is inconsistent for odd unit cells
+    #  isodd(nsites) && continue
+    #end
 
-    space_ = fill(space_shifted(model, 0; conserve_qns=conserve_qns), nsites)
+    space_ = fill(space_shifted(model, 1; conserve_qns=conserve_qns), nsites)
```

Comment:
Shouldn't it be `0` for even-site unit cells and `1` for odd-site unit cells?

Reply:
Due to the 2-modularity, for the chosen initial state, anything goes when the number of sites is even..

Reply:
Ah I see, thanks for the explanation.